js/5etools-main.js

```diff
@@ -2286,64 +2286,55 @@ const betteR205etoolsMain = function () {
 
 		function importItem (character, data, event) {
 			if (d20plus.sheet == "ogl") {
+
+				//the whole `if (data.data._versatile)` block is now obsolete because i found an easier way with "Alternate Damage"
```

Comment:
this is what Git is for--delete this junk and we can recover it from the history later, as needed. 

Also, if you're not confident in your code, then, it's not good enough to be merged.

Reply:
Tested a lot but i am new so .... in any case i am confident it is ok.

Done!